src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java

```diff
@@ -82,19 +82,29 @@ public List<TestInputViolation> getFilteredViolations() {
         return Collections.unmodifiableList(filteredViolations);
     }
 
+    public DefaultConfiguration initializeTreeWalker() {
```

Comment:
I am not seeing any external uses, so this should be private. Also please move this to be after it's usage.

Reply:
Done